include/IECorePython/RefCountedBinding.h

```diff
@@ -63,6 +63,27 @@ class RefCountedWrapper : public T, public WrapperGarbageCollector
 		template<typename Arg1, typename Arg2, typename Arg3>
 		RefCountedWrapper( PyObject *self, Arg1 arg1, Arg2 arg2, Arg3 arg3 );
 
+		template<typename Arg1, typename Arg2, typename Arg3, typename Arg4>
```

Comment:
Sure, delaying is fine. There are a number of spots where we can apply this kind of simplfication, so they could all be done in one PR. Maybe add a todo to remind us?